gateway/src/apicast/template_string.lua

```diff
@@ -0,0 +1,87 @@
+local Liquid = require 'liquid'
+local LiquidTemplate = Liquid.Template
+local LiquidInterpreterContext = Liquid.InterpreterContext
+local LiquidFilterSet = Liquid.FilterSet
+local LiquidResourceLimit = Liquid.ResourceLimit
+local ngx = ngx
+
+local setmetatable = setmetatable
+local pairs = pairs
+
+local LiquidTemplateString = {}
+local liquid_template_string_mt = { __index = LiquidTemplateString }
+
+-- Expose only ngx.* functions that we think could be useful and that do not
+-- have any side-effects.
+local liquid_filters = {
+  ngx_excape_uri = ngx.escape_uri,
```

Comment:
I'm torn on this. 

On one hand customer should not care if it is `ngx` or what. All functions to `escape_uri` should behave the same.
On the other hand it is the nginx function and in theory we might provide some other in the future.

Also, would be good to add ldoc with links to those functions so we can see it in the documentation.

Reply:
@mikz I had the same doubt while writing this code.

I'm thinking about users configuring this from 3scale's UI. I think that for them it'd be more natural to write something like 'md5' rather than 'ngx_md5'. In theory, they don't even need to know that apicast is based on nginx.

Reply:
Changed.

Reply:
Exactly 👍